### ISSUE TITLE
[Bug]Latex vector and hat symbols don't align

### ISSUE THREAD

**Issue by v5rm** [HappenApps/Quiver#905]
The \vec and \hat symbols don't align for me. I attached a screenshot of the tutorial.
<img width="1440" alt="screen shot 2017-06-26 at 17 14 53" src="https://user-images.githubusercontent.com/12582751/27565458-2a16bb3c-5a93-11e7-8f22-26a9c7f9030e.png">

**Comment by ylian** [comment]
Aware of the issue. Thanks for the report.

**Comment by Phy-David-Zhang** [comment]
A temporary solution: add the following lines in the application-wide macros under the LaTeX cell preferences

```
\renewcommand{\hat}[1]{\overset{\;\!\large\unicode{813}}{#1}}
\renewcommand{\vec}[1]{\overset{\;\!\large\unicode{8431}}{#1}}
```

and delete the lines after the update is available.

**Comment by ylian** [comment]
Fixed in 3.1.2.
